assembly/buffer/index.ts

```diff
@@ -49,3 +49,94 @@ export class Buffer extends Uint8Array {
     return load<i8>(this.dataStart + usize(offset));
   }
 }
+
+export namespace Buffer {
+  export namespace HEX {
+    /** Calculates the two char combination from the byte. */
+    @inline export function charsFromByte(byte: u32): u32 {
```

Comment:
Shouldn't the parameter be a `u8` since the function name has Byte in it and the variable is called byte?

Reply:
No. This is because it gets optimized better. @dcodeIO should be able to remember why.